tests/kafkatest/services/kafka/kafka.py

```diff
@@ -746,6 +763,18 @@ def describe_consumer_group(self, group, node=None, command_config=None):
     def zk_connect_setting(self):
         return self.zk.connect_setting(self.zk_chroot)
 
+    def connect_setting(self, node):
+        """
+        Checks if --bootstrap-server config is supported, if yes then returns a string with
+        bootstrap server, otherwise returns zookeeper connection string.
+        """
+        if node.version.supports_bootstrap_server():
+            connection_setting = "--bootstrap-server %s" % self.bootstrap_servers(self.security_protocol)
+        else:
+            connection_setting = "--zookeeper %s" % self.zk_connect_setting()
```

Comment:
Or we can support `--bootstrap-server` only. Wouldn't `node.version.supports_bootstrap_server()` always return true?

Reply:
@hachikuji  mentioned that these tests can run against a lower version of Kafka cluster, so it is possible for it to run against say 2.1 version.

Reply:
Hmmm. If that is the case then at least `kafka-leader-election.sh` won't exist in older versions of Kafka. I think we added it in 2.3.

Reply:
Talked to @hachikuji  and it seems that the test in discussion is client_compatibility_features_test.py. I looked at the test and I think you are right, we don't need older zookeeper connection. I will update the code as you suggested and post a new commit.